tests/helpers.py

```diff
@@ -240,9 +264,10 @@ def create_pvc(sc_name, pvc_name=None):
     )
     pvc_data['metadata']['namespace'] = defaults.ROOK_CLUSTER_NAMESPACE
     pvc_data['spec']['storageClassName'] = sc_name
-    return create_resource(
-        desired_status=constants.STATUS_BOUND, **pvc_data
-    )
+    ocs_obj = pvc.PVC(**pvc_data)
+    created_pvc = ocs_obj.create()
+    assert created_pvc, f"Failed to create resource {pvc_name}"
```

Comment:
This will not fail if pvc is in Pending don't we need to check status of pvc if it's bound or pending